base/libgit2/blob.jl

```diff
@@ -9,9 +9,12 @@ end
 
 Fetch the *raw* contents of the [`GitBlob`](@ref) `blob`. This is a read-only
 `Array` containing the contents of the blob, which may be binary or may be ASCII
-`String` data.
+`String` data. `rawcontent` will allow the user to load the raw binary data into
+the output `Array` and will not check to ensure it is a valid `String`, so errors
+may occur if the result is passed to functions which expect valid `String` data.
```

Comment:
is the result an array or a string? if this is type stable and always array, then you won't be passing it directly to most String functions